### PR TITLE
including wording that clarifies that offensive comments are not ok

### PR DESCRIPTION
**Pull request by paniclater**
I received some feedback that the current CoC might make it seem like someone organizing a coding group for women would feel like that goal could be against the CoC as it stands.

I revisited the react conf CoC that inspired the change originally and noticed they include a very important word that we don't currently have: offensive.

>Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces

This change adds the word offensive to the clarification on appropriate verbal or written comments.

### CODE REVIEW

**Comment by maddie927** [comment]
"Offensive" might even be a little vague. It's definitely correct, but I worry about remarks like, "Well, it offends me that I can talk the way I want here!"

Perhaps:

> Verbal or written comments which degrade or endanger a protected group, or an individual in the context of that individual being a member of a protected group. Protected groups include gender identity or expression, sexual orientation, disability, physical appearance, body size, race and religion.

**Comment by ryoia** [comment]
Agreed

**Comment by paniclater** [comment]
What about:

> Verbal or written comments which degrade or endanger an an individual in the context of gender identity or expression, sexual orientation, disability, physical appearance, body size, race and religion.

It captures the spirit of clarifying the context of what offensive comments are specifically innapropropriate without referencing protected groups. I'm a little bit antsy about using the term protected group specifically because Utah law to my knowledge does not include gender identity or expression within protected groups. 

Definitely open to including it still, just some thoughts.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Examples of unacceptable behavior by participants include:
 - The use of sexualized language or imagery and unwelcome sexual attention or advances
 - Trolling, insulting/derogatory comments, and personal or political attacks
 - Public or private harassment
-- Verbal or written comments related to gender, sexual orientation, disability, physical appearance, body size, race or religion
+- Verbal or written offensive comments related to gender, sexual orientation, disability, physical appearance, body size, race or religion
```
Comment (kensnyder) [inline README.md]:
```suggestion
- Harassment or insensitivity related to gender, sexual orientation, disability, physical appearance, body size, race or religion
```
What about this change?

**Comment by maddie927** [comment]
Utah actually does include gender identity, surprisingly. But I see what you mean. Ken's latest suggestion seems pretty good, since it's just as simple but isn't as vague.

**Comment by maddie927** [comment]
I also was thinking the definition of "protected groups" would be our own, not another organization's or the state's.

**Comment by paniclater** [comment]
That is really good to hear about gender identity in Utah. I'll update the text w/ Ken's suggestion. Thanks!

**Comment by paniclater** [comment]
Ken's changes are applied. 🎉

**Comment by maddie927** [comment]
Proposed one more tweak (in the commit above since I already discussed it with @paniclater)

**Comment by maddie927** [comment]
Going to merge this since it seemed like everyone was in agreement. We can always open another PR.

**Pull request merged**